source-formatter.properties

```diff
@@ -545,6 +545,7 @@
         com.liferay.portal.kernel.xmlrpc.XmlRpcUtil
         com.liferay.portal.kernel.zip.ZipReaderFactoryUtil
         com.liferay.portal.kernel.zip.ZipWriterFactoryUtil
+        com.liferay.sites.kernel.util.SitesUtil
```

Comment:
mistake is here

Reply:
Hey @julschong 

That is an interesting issue.

I did some tests for it, I think SF is not smart enough to know how to check it.
Let me show some examples to explain why we can't add SF logic to check it.

----------------
_Example 1:_
```
key=a\
    b
```
The value of `key` is `ab`.

-------------------
_Example 2;_
```
key=a
    b
```
Oh, now we will have two keys.
the value of `key` is `a`, and the value of `b` is `empty` even we didn't add `=` for `b`.

-------------------
BTW, I use `java.util.Properties` to load properties.

Let me know what you think about it.

Reply:
I'm just thinking showing examples to explain it is better than my bad English.

Reply:
that is very interesting :smile: I'll let Tina know tomorrow and see if we are ok